### PR TITLE
refactor(experimental): graphql: token-2022 extensions: ApproveConfidentialTransferAccount, EmptyConfidentialTransferAccount

### PR DESCRIPTION
**Pull request by Hrushi20**
This PR adds support for Token-2022's ApproveConfidentialTransferAccount, EmptyConfidentialTransferAccount instructions in the GraphQL schema.

Continuing work on https://github.com/solana-labs/solana-web3.js/issues/2406.

### CODE REVIEW

File: packages/rpc-graphql/src/__tests__/__setup__.ts [hunk 1]
```diff
@@ -2079,6 +2079,51 @@ export const mockTransactionToken2022AllExtensions = {
...
+                    },
+                    program: 'spl-token',
+                    programId: 'TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb',
+                    stackHeight: null,
+                },
+                {
+                    parsed: {
+                        info: {
+                            account: '2Pwe6Yahh5cbzvCwRMtTYFeboSwYiWeHhYJzZZBsU6eB',
+                            instructionsSysvar: 'SysvarRent111111111111111111111111111111111',
```
Comment (buffalojoec) [inline packages/rpc-graphql/src/__tests__/__setup__.ts:2099]: I know it's just a dummy value for tests, but fyi this is always going to be `Sysvar1nstructions1111111111111111111111111`.

**Review by buffalojoec — APPROVED** [review]
Looks great! Thanks for the continued effort!

File: packages/rpc-graphql/src/__tests__/__setup__.ts (hunk 1 above)
Comment (Hrushi20) [reply packages/rpc-graphql/src/__tests__/__setup__.ts:2099]: Should I update it?

Comment (buffalojoec) [reply packages/rpc-graphql/src/__tests__/__setup__.ts:2099]: Yeah, why not!

**Pull request merged**